### ISSUE TITLE
Documentation on creating EKS ingress

### ISSUE THREAD

**Issue by virmundi** [rancher/rancher#18706]
**What kind of request is this (question/bug/enhancement/feature request):**
Enhancement

**Steps to reproduce (least amount of steps as possible):**
1. Create new EKS cluster.
1. Get warning message at start "	
Note: Currently Amazon EKS will not create an ingress controller when launching a new cluster. If you need this functionality you will have to create an ingress controller manually after cluster creation."
1. Create cluster

**Result:**
A cluster exists, but is not accessible.

Could you please add documentation on how to setup EKS' load balancer? https://rancher.com/docs/rancher/v2.x/en/k8s-in-rancher/load-balancers-and-ingress/ doesn't help. It leaves the entire process as hard to follow exercise.

**Comment by dnoland1** [comment]
Does the documentation at https://rancher.com/docs/rancher/v2.x/en/k8s-in-rancher/load-balancers-and-ingress/ingress/ which has step-by-step directions on how to create an ingress controller answer your question?

**Comment by dnoland1** [comment]
This blog on ingress and EKS may also be helpful - https://rancher.com/blog/2018/2018-06-08-load-balancing-user-apps-with-rancher/

**Comment by neilfrndes** [comment]
@virmundi was your question about how expose the NGINX Ingress controller behind a Service of Type=LoadBalancer in AWS using an Elastic Load Balancer (ELB). https://kubernetes.github.io/ingress-nginx/deploy/#aws

**Comment by superseb** [comment]
If you are using hosted Kubernetes, it's common to use the options from the hoster for features. In case of EKS, the options for load balancing and ingress are described at https://docs.aws.amazon.com/eks/latest/userguide/load-balancing-and-ingress.html. Putting all the information in our own docs will only lead to outdated information as the hoster might change recommendations/options and we have to copy paste that into our own docs. The main page on https://rancher.com/docs/rancher/v2.x/en/cluster-provisioning/hosted-kubernetes-clusters/eks/ also links to the main page of the EKS docs.

I created https://github.com/rancher/docs/issues/1649 to revise this page to explain more what the options are and where to look for more info.
